support-frontend/assets/helpers/productPrice/subscriptions.ts

```diff
@@ -29,12 +30,13 @@ export type ComponentAbTest = {
 	name: string;
 	variant: string;
 };
-type TrackingProperties = {
+export type TrackingProperties = {
```

Comment:
This type is now exported so that it can be used over in `paperPrices.tsx`, which addresses a couple of parameter type errors.